apps/guide/settings/base.py

```diff
@@ -184,4 +238,228 @@
 WAGTAILIMAGES_IMAGE_MODEL = "custom_media.CustomImage"
 WAGTAILDOCS_DOCUMENT_MODEL = "custom_media.CustomDocument"
 
+pixel_limit = env.get("WAGTAILIMAGES_MAX_IMAGE_PIXELS")
+WAGTAILIMAGES_MAX_IMAGE_PIXELS = int(pixel_limit) if pixel_limit else 10_000_000
+
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+# AWS S3 buckets configuration
+# This is media files storage backend configuration. S3 is our preferred file
+# storage solution.
+# To enable this storage backend we use django-storages package...
+# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html
+# ...that uses AWS' boto3 library.
+# https://boto3.amazonaws.com/v1/documentation/api/latest/index.html
+#
+# Three required environment variables are:
+#  * AWS_STORAGE_BUCKET_NAME
+#  * AWS_ACCESS_KEY_ID
+#  * AWS_SECRET_ACCESS_KEY
+# The last two are picked up by boto3:
+# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variables
+if "AWS_STORAGE_BUCKET_NAME" in env:
+    # Add django-storages to the installed apps
+    INSTALLED_APPS = INSTALLED_APPS + ["storages", "wagtail_storages"]
```

Comment:
wagtail_storages is only needed if we're to have private documents. Given this is a public project, is it really needed?

Reply:
I am still nitpicking on this as I don't see the need for it.
less stuff to install/load, the better :P

Reply:
Added as an issue to follow up on later: https://github.com/wagtail/guide/issues/54.